sections/cart-notification-product.liquid

```diff
@@ -20,6 +20,23 @@
                 <dd>{{ option.value }}</dd>
               </div>
             {%- endfor -%}
+            {%- for property in item.properties -%}
+              {%- assign property_first_char = property.first | slice: 0 -%}
+              {%- if property.last != blank and property_first_char != '_' -%}
+                <div class="product-option">
+                  <dt>{{ property.first }}: </dt>
```

Comment:
I wonder if we should ask the translation team about the use of `:`. Does it mean the same thing everywhere ? 🤔

Reply:
Good question. We actually use colon the same way in a few areas of the theme, I would be curious to know how best to handle this for i18n